t/full-stack.t

```diff
@@ -525,5 +527,15 @@ END {
     kill_driver;
     turn_down_stack;
     session->clean;
+    # prevent double execution
+    return if $? == 0;
     $? = 0;
+    find(
+        sub {
+            if (-f and /\.txt$/) {
+                print "$_:\n" . path($_)->slurp;
+            }
+        },
+        $tempdir
+    );
```

Comment:
How about a one-liner instead? The `list_tree` method only includes files by default (no directories).
```perl
$tempdir->list_tree->grep(qr/\.txt$/)->each(sub { print "$_:\n" . $_->slurp });
```
This combines features from [Mojo::File](https://mojolicious.org/perldoc/Mojo/File) and [Mojo::Collection](https://mojolicious.org/perldoc/Mojo/Collection). Both are really nice for making Perl more readable.

Reply:
nice, I will use that :)